src/common/arrow/arrow_converter.cpp

```diff
@@ -79,7 +79,7 @@ void ArrowConverter::setArrowFormat(
     case DataTypeID::STRING: {
         child.format = "u";
     } break;
-    case LIST: {
+    case VAR_LIST: {
```

Comment:
Can you open an issue that we should support export of FIXED LIST to arrow table?

Reply:
opened issue #1310